geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,76 @@
 from django.test import TestCase
+from django.conf import settings
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument, Parameter
+from geospaas.vocabularies.models import ISOTopicCategory, DataCenter
+from geospaas.catalog.models import *
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
+
+    fixtures = ["vocabularies", "catalog"]
+
+    def test_api_initial_calls(self):
```

Comment:
It is better to have a separate test for each GET request. Then tests are run independently. And if one GET-request fails - then the test will fail and it will show exactly which GET request failed. In the implementation below if one GET-request fails, then other requests are not tested.
TODO: Split one method into separate methods for each GET-request.